docs/static/jvm.asciidoc

```diff
@@ -53,8 +53,7 @@ Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 variable must be set for {ls} to operate correctly.
 
 If {ls} doesn't find `LS_JAVA_HOME` it tries to fall back to `JAVA_HOME`.
-Note that this fallback behavior is experimental and the usage of `JAVA_HOME`
-could be subject to removal in later releases.
+The usage of `JAVA_HOME` is considered deprecated.
```

Comment:
The usage of `JAVA_HOME` is now considered deprecated in favor of `LS_JAVA_HOME`.